src/main/generated/resources/data/minecolonies/crafterrecipes/sifter/diamond/dirt.json

```diff
@@ -6,12 +6,12 @@
       "item": "minecraft:dirt"
     },
     {
-      "item": "minecolonies:sifter_mesh_diamond"
+      "item": "minecolonies:sifter_mesh_diamond{Damage:0}"
```

Comment:
This worries me, as we don't want to match to Damage:0 here.

Reply:
It was added due to the automatic support for any NBT values, and that being the default for a fresh ItemStack.

I was a bit worried about it at first as well, but I've tested it with a sifter in-world and they'll still happily do crafting operations with damaged meshes, so the code is overriding things in the right ways.  (But it would have had to, because specifying the item without NBT would have never matched any real-world item either.)

So the only effect this actually has is on the preview item, but there's no visual difference and it's the same as what it would default to the instant you put it into an ItemStack anyway.  So I think it's fine as is, other than producing a little clutter in the JSON.

Reply:
Ok.